Integrations/integration-Carbon_Black_Enterprise_Response.yml

```diff
@@ -43,6 +43,11 @@ configuration:
   defaultvalue: ""
   type: 0
   required: false
+- display: Max Incidents Per Fetch
```

Comment:
I think that it should be something like `The maximum number of incidents to fetch` but need to consult with @kirbles19 about this
FYI build this example using https://confluence.paloaltonetworks.com/display/DemistoContent/Integrations+And+Scripts

Reply:
Maximum number of incidents to fetch
OR
Maximum number of incidents per fetch